Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,62 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
+
+## Organization Roles
+
+### Owners
+
+The Node.js TSC Director, TSC Facilitator, and the two elected individual
+member representatives shall be granted Owner permissions within the Node.js
+GitHub Organization.
+
+Additional TSC, CommComm, and Core Technical Committee (CTC) members may be
+granted Owner permissions by request. Assuming there are no objections from any
+current voting TSC or CommComm member, such requests are approved automatically
+after 72 hours. If any objection is made, the request may be moved to a single
+vote in which a simple majority of both the TSC and CommComm is required to
+approve. Such requests must be posted as issues to either the nodejs/TSC or
+nodejs/community-committee repositories.
```

Comment:
Is the idea here that people would request access via the TSC or CommComm based on who is closer to their area? e.g. a CTC or TSC member would request through the TSC, while a CommComm member would request through CommComm? Should the other committee be notified when one committee receives a request? (I can see arguments both ways)

I don't think we should formalize any of this in this doc (I'd prefer to keep it more generic and flexible), I just want to make sure I understand the idea behind it.